src/pages/2020_2021.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
```

Comment:
Remove unused components in import

Reply:
Hey this was the page which was commited before..Actually this page is not used.I didnt know how to remove that.Sorry.